src/apiFunctions.js

```diff
@@ -0,0 +1,13 @@
+const getPokemon = () => fetch('https://pokeapi.co/api/v2/pokemon/?limit=20&offset=0/').then((response) => response.json());
+
+const arrayPokemonLinks = async () => {
+  const data = await getPokemon();
+  const array = data.results;
```

Comment:
maybe you can consider using `array = await getPokemon().results` to reduce the number of variables used.

Reply:
https://github.com/MacCrazyman/pokemon-capstone/blob/8ea361656ee9ae4d7370344f4baec26f3ee94da4/src/apiFunctions.js#L4

Good job implementing this feature 👍